tasks/collect-data/README.md

```diff
@@ -20,10 +20,13 @@ also a task result for the fbcFragment extracted from the snapshot's first compo
 | release              | Namespaced name of the Release                     | No       | -             |
 | releaseplan          | Namespaced name of the ReleasePlan                 | No       | -             |
 | releaseplanadmission | Namespaced name of the ReleasePlanAdmission        | No       | -             |
-| releasestrategy      | Namespaced name of the ReleaseStrategy             | No       | -             |
 | snapshot             | Namespaced name of the Snapshot                    | No       | -             |
 | subdirectory         | Subdirectory inside the workspace to be used.      | Yes      | -             |
 
+## Changes since 0.4
+  * Remove ReleaseStrategy
+  * Collect Data insteand of ExtraData
```

Comment:
typo

Reply:
Can you be more descriptive here? Like that we are grabbing a different field in the spec due to API changes and the task result has changed

Reply:
Done.